build.gradle

```diff
@@ -249,7 +249,9 @@ task createAboutText(type: Task) {
     def projectUrlText = readmeContentArray.subList(projectUrlStart, projectUrlEnd).join(" ")
     def gitUrl = projectUrlText.substring(projectUrlText.indexOf("https"))
     gitUrl = gitUrl.substring(0, gitUrl.indexOf("releases"))
-    final def aboutJson = JsonOutput.toJson([version: version, projectUrl: gitUrl, description: description])
+
+    def time = new Date().format('yyy-MM-dd\'T\'HH:mm:ss.SSSSSS\'Z\'');
```

Comment:
We should use `UpdateChecker. DATE_FORMAT` here.

Reply:
I don't believe we can reference the class from the gradle file. If you know how to do that please let me know

Reply:
Oh, you're right. I think you can only do it with buildscript dependencies.